servatrice/scripts/linux/maint_replays

```diff
@@ -1,8 +1,9 @@
 #!/bin/bash
 
-# SCHEDULE WITH CRONTAB BASED ON TIME PERIOD REPLAYS SHOULD BE SAVED UNTIL (EX: SCHEDULE ONCE A WEEK TO KEEP A WEEKS WORTH OF REPLAYS IN THE DB)
-
+# SCHEDULE WITH CRONTAB DAILY
 DBNAME="servatrice"					#set this to the database name used
 TABLEPREFIX="cockatrice"			#set this to the prefix used for the table names in the database (do not inclue the _)
 SQLCONFFILE="./mysql.cnf" 			#set this to the path that contains the mysql.cnf file
-mysql --defaults-file=$SQLCONFFILE -h localhost -e 'truncate table ""$DBNAME"".""$TABLEPREFIX""_replays;truncate table ""$DBNAME"".""$TABLEPREFIX""_replays_access'
+mysql --defaults-file=$SQLCONFFILE -h localhost -e 'delete from servatrice.cockatrice_games where time_finished < DATE_SUB(now(), INTERVAL 8 DAY)'
+mysql --defaults-file=$SQLCONFFILE -h localhost -e 'delete from servatrice.cockatrice_replays where id_game NOT IN (select id from servatrice.cockatrice_games)'
+mysql --defaults-file=$SQLCONFFILE -h localhost -e 'delete from servatrice.cockatrice_replays_access where id_game NOT IN (select id from servatrice.cockatrice_games)'
```

Comment:
I'm OK with doing it this script-based way for now but I'd like to move to an in-db option later.

How about 

delete_old_replays.sql
```sql
START TRANSACTION;
DROP TEMPORARY TABLE IF EXISTS old_games;
CREATE TEMPORARY TABLE old_games AS (SELECT id FROM cockatrice_games WHERE time_finished < DATE_SUB(now(), INTERVAL 8 DAY));

DELETE FROM cockatrice_replays r JOIN old_games g ON r.id_game = g.id;
DELETE FROM cockatrice_replays_access ra JOIN old_games g ON ra.id_game = g.id;
DELETE FROM cockatrice_games g JOIN old_games og ON g.id = og.id;

DROP TEMPORARY TABLE old_games;
COMMIT;
```